.github/workflows/main.yml

```diff
@@ -94,51 +115,6 @@ jobs:
         if: ${{ env.DEPLOY_ENV == 'Production' }}
         run: echo 'DEPLOY_URL=${{ steps.deploy-to-vercel-production.outputs.preview-url }}' >> "$GITHUB_ENV"
 
-      - name: Initialize GitHub Deployment
```

Comment:
we never used the github deployments feature and don't use them in the content repos too